_tutorials/2016-05-10-iosxr-ansible-md.md

```diff
@@ -129,7 +128,7 @@ vagrant@localhost's password:
 RP/0/RP0/CPU0:ios#
 ```
 
-Now let's  configure IP address at XR box. Issue the commant at XR CLI.
+Now let's  configure IP address at IOS-XRv box. Issue the command at XR CLI.
```

Comment:
Replace this sentence with:
Now let's configure an IP address on the IOS-XRv instance. Issue the following command on the XR cli: